src/lib/utils/options/readers/tsconfig.ts

```diff
@@ -82,7 +82,7 @@ export class TSConfigReader extends OptionsComponent
             delete compilerOptions[key];
         }
 
-        _.merge(event.data, compilerOptions);
+        _.defaults(event.data, compilerOptions);
 
         if ("typedocOptions" in data) {
             _.merge(event.data, data.typedocOptions);
```

Comment:
Even better! Thanks for all the feedback. 

p.s. I confirmed that `_.defaults` accepts `undefined` args in case `data.raw.typedocOptions` is undefined.